admin/action/modules/fman.php

```diff
@@ -320,7 +322,43 @@
                 }
                 break;
 
-                // odstraneni vyberu
+            // stazeni souboru
+            case "downloadselected":
+                // ziskani vybranych souboru
+                $selected = array();
+                foreach ($_POST as $var => $val) {
+                    if ($var == "action" || $var == "param") {
+                        continue;
+                    }
+                    $val = $decodeFilename($val);
+                    if (file_exists($dir . $val) && !is_dir($dir . $val) && User::checkFilename($val)) {
```

Comment:
`file_exists() && !is_dir()` can be replaced by a single call to `is_file()`

Reply:
I just wanted to keep CS.

https://github.com/sunlight-cms/sunlight-cms-8/blob/master/admin/action/modules/fman.php#L333

Reply:
I guess leave it be then, unless you want to change it everywhere 😄

Reply:
Well, I don't want to change it everywhere ...

Reply:
Done